_posts/2021-04-26-changelog-146.markdown

```diff
@@ -0,0 +1,190 @@
+  (see [#7094](https://github.com/intellij-rust/intellij-rust/pull/7094) for more details)
+
+* [#6903] Introduce `2021` edition variant in the plugin and adjust the corresponding code
+
+Full set of changes can be found [here](https://github.com/intellij-rust/intellij-rust/milestone/54?closed=1)
+
+[@Kobzol]: https://github.com/Kobzol
+[@Stzx]: https://github.com/Stzx
+[@abn]: https://github.com/abn
+
+[#7142]: https://github.com/intellij-rust/intellij-rust/pull/7142
+[#7112]: https://github.com/intellij-rust/intellij-rust/pull/7112
+[#7111]: https://github.com/intellij-rust/intellij-rust/pull/7111
+[#7100]: https://github.com/intellij-rust/intellij-rust/pull/7100
+[#7093]: https://github.com/intellij-rust/intellij-rust/pull/7093
+[#7071]: https://github.com/intellij-rust/intellij-rust/pull/7071
+[#6992]: https://github.com/intellij-rust/intellij-rust/pull/6992
+[#6986]: https://github.com/intellij-rust/intellij-rust/pull/6986
+[#6960]: https://github.com/intellij-rust/intellij-rust/pull/6960
+[#6845]: https://github.com/intellij-rust/intellij-rust/pull/6845
+[#6837]: https://github.com/intellij-rust/intellij-rust/pull/6837
+[#6801]: https://github.com/intellij-rust/intellij-rust/pull/6801
+[#6599]: https://github.com/intellij-rust/intellij-rust/pull/6599
+[#6013]: https://github.com/intellij-rust/intellij-rust/pull/6013
+[#5650]: https://github.com/intellij-rust/intellij-rust/pull/5650
+[#5491]: https://github.com/intellij-rust/intellij-rust/pull/5491
+[#7150]: https://github.com/intellij-rust/intellij-rust/pull/7150
+[#7132]: https://github.com/intellij-rust/intellij-rust/pull/7132
+[#7131]: https://github.com/intellij-rust/intellij-rust/pull/7131
+[#7121]: https://github.com/intellij-rust/intellij-rust/pull/7121
+[#7119]: https://github.com/intellij-rust/intellij-rust/pull/7119
+[#7115]: https://github.com/intellij-rust/intellij-rust/pull/7115
+[#7108]: https://github.com/intellij-rust/intellij-rust/pull/7108
+[#7101]: https://github.com/intellij-rust/intellij-rust/pull/7101
+[#7089]: https://github.com/intellij-rust/intellij-rust/pull/7089
+[#7081]: https://github.com/intellij-rust/intellij-rust/pull/7081
+[#7080]: https://github.com/intellij-rust/intellij-rust/pull/7080
+[#7079]: https://github.com/intellij-rust/intellij-rust/pull/7079
+[#7076]: https://github.com/intellij-rust/intellij-rust/pull/7076
+[#7072]: https://github.com/intellij-rust/intellij-rust/pull/7072
+[#7070]: https://github.com/intellij-rust/intellij-rust/pull/7070
+[#7057]: https://github.com/intellij-rust/intellij-rust/pull/7057
+[#7004]: https://github.com/intellij-rust/intellij-rust/pull/7004
+[#6201]: https://github.com/intellij-rust/intellij-rust/pull/6201
+[#7114]: https://github.com/intellij-rust/intellij-rust/pull/7114
+[#7094]: https://github.com/intellij-rust/intellij-rust/pull/7094
+[#7046]: https://github.com/intellij-rust/intellij-rust/pull/7046
+[#6903]: https://github.com/intellij-rust/intellij-rust/pull/6903
```

Comment:
I'd personally sort these lines but it's up to you.
`Sort lines` action can help here